### PR TITLE
Improve installation instructions in the README

### PR DESCRIPTION
**Pull request by AlbydST**
I added installation instructions for
- Exherbo
- GNU Guix
- Solus
- Slackware
- Void Linux
- RebornOS
- Chocolatey on Windows

I also improved installation instructions for Termux and changed `sudo` to `su -c` in commands, this executes the command as root regardless of the presence of sudo / doas / sup.

### CODE REVIEW

**Comment by CarterLi** [comment]
> * RebornOS

I don't think installing an ancient version is a good idea.

> changed `sudo` to `su -c` in commands, this executes the command as root regardless of the presence of sudo / doas / sup

I don't like this

**Comment by AlbydST** [comment]
> I don't think installing an ancient version is a good idea.

Should I specify that it's outdated or remove it entirely?

> I don't like this

Can you explain why?

**Comment by CarterLi** [comment]
> Can you explain why?

They are different. For example, `su` requires root user's password, while `sudo` requires current user's.

**Comment by CarterLi** [comment]
> Should I specify that it's outdated or remove it entirely?
> 

Remove it

File: README.md [hunk 1]
```diff
@@ -30,17 +30,22 @@ There are [screenshots on different platforms](https://github.com/fastfetch-cli/
...
-* Gentoo: `sudo emerge --ask app-misc/fastfetch`
-* Alpine: `apk add --upgrade fastfetch`
-* NixOS: `nix-shell -p fastfetch`
-* openSUSE: `sudo zypper install fastfetch`
-* ALT Linux: `sudo apt-get install fastfetch`
-
-Replace sudo with doas depending on what you use.
+* Arch Linux: `su -c pacman -Syy fastfetch`. An outdated version of fastfetch is also available [on the AUR](https://aur.archlinux.org/packages/fastfetch-git).
+* Fedora: `su -c dnf install fastfetch`
+* Gentoo: `su -c emerge --ask --verbose app-misc/fastfetch`
```
Comment (CarterLi) [inline README.md:37]: Why `--verbose`?

File: README.md [hunk 2]
```diff
@@ -30,17 +30,22 @@ There are [screenshots on different platforms](https://github.com/fastfetch-cli/
...
+* Alpine: `su -c apk add --upgrade fastfetch`
+* NixOS: `su -c nix-env -iA nixos.fastfetch`
```
Comment (CarterLi) [inline README.md:39]: Why this change?

File: README.md [hunk 3]
```diff
@@ -30,17 +30,22 @@ There are [screenshots on different platforms](https://github.com/fastfetch-cli/
...
+* openSUSE: `su -c zypper install fastfetch`
+* ALT Linux: `su -c apt-get install fastfetch`
+* Exherbo: `su -c cave resolve --execute app-misc/fastfetch`
+* GNU Guix: `su -c guix install fastfetch`
+* Solus: `su -c eopkg install fastfetch`
+* Slackware: With [sbopkg](https://www.sbopkg.org/index.php), run `su -c sbopkg -i fastfetch` to install [the fastfetch SlackBuild](https://slackbuilds.org/repository/15.0/system/fastfetch/).
```
Comment (CarterLi) [inline README.md:45]: Please remove `With ***` `to install ***`

File: README.md [hunk 4]
```diff
@@ -65,7 +71,10 @@ You may also download the program directly from [the GitHub releases page](https
 
 ### Android (Termux)
 
-* `pkg install fastfetch`
+* Fastfetch is available [in the Termux repository](https://github.com/termux/termux-packages/tree/master/packages/fastfetch), run `pkg install fastfetch`
+
+* You can also install fastfetch directly with APT: `apt install fastfetch`
+
```
Comment (CarterLi) [inline README.md:77]: Don't change this one.

File: README.md [hunk 5]
```diff
@@ -103,7 +112,7 @@ Logos can be heavily customized too; see the [logo documentation](https://github
 
 1. Fastfetch is actively maintained.
 2. Fastfetch is faster. As the name suggests.
-3. Fastfetch has a greater number of features, though by default fastfetch only has a few modules enabled; use `fastfetch -c all` to find what you want.
+3. Fastfetch has a greater number of features, though by default fastfetch only has a few modules enabled; use `fastfetch -c all.jsonc` to find what you want.
```
Comment (CarterLi) [inline README.md:115]: Revert this

File: README.md [hunk 6]
```diff
@@ -112,7 +121,7 @@ Logos can be heavily customized too; see the [logo documentation](https://github
 
 A local IP (10.x.x.x, 172.x.x.x, 192.168.x.x) has nothing to do with privacy. It only makes sense if you are on the same network, for example, if you connect to the same Wi-Fi network.
 
-Actually the `Local IP` module is the most useful module for me personally. I (@CarterLi) have several VMs installed to test fastfetch and often need to SSH into them. I have fastfetch running on shell startup and I never need to type `ip addr` manually.
+Actually the `Local IP` module is the most useful module for me personally. I ([@CarterLi](https://github.com/CarterLi)) have several VMs installed to test fastfetch and often need to SSH into them. I have fastfetch running on shell startup and I never need to type `ip addr` manually.
```
Comment (CarterLi) [inline README.md:124]: Revert this
